public/Day 02/index.html

```
@@ -1,290 +1,141 @@
+        <div><h3>10K+</h3><p>Users</p></div>
+        <div><h3>99.9%</h3><p>Uptime</p></div>
+        <div><h3>24/7</h3><p>Support</p></div>
       </div>
     </div>
 
-
-    <div class="hero-image">
-      <img src="https://images.unsplash.com/photo-1551288049-bebda4e38f71"
-           alt="Technology">
+    <div class="hero-img">
+      <img src="https://images.unsplash.com/photo-1551288049-bebda4e38f71" alt="">
```

HTMLHint (1.9.1)

[warning] The empty tag : [ img ] must be self closed.

(tag-self-close)